nengo/utils/tests/test_progress_file

```diff
@@ -0,0 +1 @@
+0%, ETA: 0:00:00


```

Comment:
Also committed accidentally I assume.

Reply:
If these get generated as part of testing the progress bar, we should add these to `.gitignore` also.

Reply:
I opted to create the file on-the-fly in the test itself.